tests/integ/test_column_names.py

```diff
@@ -417,18 +436,51 @@ def add_one(x: int) -> int:
     perm_func_name = (
         f'"{special_chars}{Utils.random_name_for_temp_object(TempObjectType.FUNCTION)}"'
     )
+    database_name = session.get_current_database()
+    schema_name = session.get_current_schema()
+    full_temp_func_name = f"{database_name}.{schema_name}.{temp_func_name}"
+
     stage_name = Utils.random_stage_name()
-    try:
+    if not local_testing_mode:
         Utils.create_stage(session, stage_name, is_temporary=False)
-        df = session.create_dataframe([1, 2], schema=["a"])
+
+    df = session.create_dataframe([1, 2], schema=["a"])
+
+    try:
+        if not local_testing_mode:
+            # Local Testing does not support permanent registration of extension functions
```

Comment:
I believe there's a ticket for adding that support. Could you annotate this comment with the ticket number?

Reply:
is_permanent should not be supported. Permanent objects should be accessible across different session objects, which is not the case in Local Testing since everything is session scoped.